crates/oxc_semantic/src/scope.rs

```diff
@@ -74,6 +74,11 @@ impl ScopeTree {
         self.parent_ids[scope_id]
     }
 
+    /// Get a variable binding by name that was declared in the top-level scope
+    pub fn get_root_binding(&self, name: &Atom) -> Option<SymbolId> {
```

Comment:
shouldn't this just be inlined rather than a whole new function if it's only used once?

Reply:
Could be, I thought it might be useful to other devs. Do you think it could be useful later on?